modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/hemophage.dm

```diff
@@ -87,5 +87,7 @@
 
 /datum/species/hemophage/prepare_human_for_preview(mob/living/carbon/human/human)
 	human.skin_tone = "albino"
+	human.hair_color = "#000000"
+	human.hairstyle = "Long Sideparted"
```

Comment:
Just for aesthetic reasons, I think we should probably try and go with a hairstyle that covers the face a bit less. I can go and look for one and hit you up on Discord with a suggestion for a replacement, if you'd like.

Reply:
I have no clue what hairstyles really exist that'd fit the bill- someone did mention Pompadour, though I haven't checked it yet.